QHyper/converter.py

```diff
@@ -210,12 +183,12 @@ def binary_to_discrete(v: str) -> str:
                 dqm.set_quadratic(
                     dqm.variables[xi_idx],
                     dqm.variables[xj_idx],
-                    {(case, case): bias for case in range(problem.cases + CASES_OFFSET)},
+                    {(case, case): bias for case in range(problem.cases + cases_offset)},
                 )
             else:
                 dqm.set_linear(
                     dqm.variables[xi_idx],
-                    [bias for _ in range(problem.cases + CASES_OFFSET)],
+                    [bias for _ in range(problem.cases + cases_offset)],
```

Comment:
I think that will work too
```suggestion
                    [bias] * (problem.cases + cases_offset),
```